packages/core/rollup.config.npm.mjs

```diff
@@ -15,7 +15,7 @@ const plugins = [
 ]
 
 const external = [
-  // '@bugsnag/cuid',
+  '@bugsnag/cuid',
```

Comment:
In the jest tests imports from `@bugsnag/core/breadcrumb` refer to the source code and changing this to `import { Breadcrumb } from '@bugsnag/core'` now references the built dist code. Jest blew up at this point as it parses the whole cjs bundle and sees undefined globals like `navigator` and `self`.

I think core should really externalise all its dependencies and it be left to the individual notifiers to decide what, if any node modules to externalise.